tests/Mautic/StageBundle/Tests/Entity/StageRepositoryTest.php

```diff
@@ -1,137 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Mautic\StageBundle\Tests\Entity;
-
-use Doctrine\ORM\AbstractQuery;
-use Doctrine\ORM\EntityManager;
-use Doctrine\ORM\Mapping\ClassMetadata;
-use Doctrine\ORM\QueryBuilder;
-use Mautic\StageBundle\Entity\Stage;
-use Mautic\StageBundle\Entity\StageRepository;
-
-class StageRepositoryTest extends \PHPUnit\Framework\TestCase
```

Comment:
This one doesn't seem to be related to Codeception. I realize now that tests like this one are not helpful and a short functional test would do a better job. But do we want to throw it away?

Reply:
that test hasn't been executed for a long time, as it's not on a folder that phpunit checks.

see the number of tests before and after that commit

[before](https://github.com/mautic/mautic/actions/runs/4743304878/jobs/8422682201):

```
OK, but incomplete, skipped, or risky tests!
Tests: 3325, Assertions: 9967, Skipped: 5.
```

[after](https://github.com/mautic/mautic/actions/runs/4746183342/jobs/8429482783)
```
OK, but incomplete, skipped, or risky tests!
Tests: 3325, Assertions: 9967, Skipped: 5.
```

Reply:
So re-adding it would be another PR, tis is the cleanup

Reply:
Ah, I see. It's in a wrong folder. I'm OK with removing it then. It tests implementation and not I/O so it's close to useless anyway.